src/components/gltf-model-plus.js

```diff
@@ -204,15 +204,19 @@ const inflateEntities = function(indexToEntityMap, node, templates, isRoot, mode
 };
 
 function inflateComponents(inflatedEntity, indexToEntityMap) {
-  inflatedEntity.object3D.traverse(object3D => {
+  let previousInflation;
+  inflatedEntity.object3D.traverse(async object3D => {
+    if (previousInflation) {
+      await previousInflation;
+    }
     const entityComponents = getHubsComponents(object3D);
     const el = object3D.el;
 
     if (entityComponents && el) {
       for (const prop in entityComponents) {
         if (entityComponents.hasOwnProperty(prop) && AFRAME.GLTFModelPlus.components.hasOwnProperty(prop)) {
           const { componentName, inflator } = AFRAME.GLTFModelPlus.components[prop];
-          inflator(el, componentName, entityComponents[prop], entityComponents, indexToEntityMap);
+          previousInflation = inflator(el, componentName, entityComponents[prop], entityComponents, indexToEntityMap);
```

Comment:
what raised this issue?

Reply:
I didn't run into an issue, but I didn't want to break the contract where inflation is currently sequential, given we often have surprises with component ordering and instantiation

Reply:
Actually, this code is probably wrong, now that I look at it. I'll do more proper testing later, but I do still want to maintain that contract.